config/environments/development.rb

```diff
@@ -6,7 +6,7 @@
   config.cache_classes = !!Sidekiq.server?
 
   # Verifies that versions and hashed value of the package contents in the project's package.json
-  config.webpacker.check_yarn_integrity = true
+  config.webpacker.check_yarn_integrity = false
```

Comment:
Should we add the dual boot conditional to these? App.rails_5_1?

Reply:
it should always be false. it has to do with a buggy yarn command